tests/serializers/test_json.py

```diff
@@ -80,6 +81,19 @@ def test_indentation_whitespace(self):
             if re.search(r'.+,\s*$', line):
                 self.assertEqual(line, line.rstrip())
 
+    def test_custom_encoder(self):
+        class CustomJSONEncoder(DjangoJSONEncoder):
+            def default(self, o):
+                if isinstance(o, decimal.Decimal):
+                    return str(o)
+
+        ScoreDecimal.objects.create(score=decimal.Decimal(1.0))
+        qset = ScoreDecimal.objects.all()
+
+        s = serializers.json.Serializer()
+        json_data = s.serialize(qset, cls=CustomJSONEncoder)
```

Comment:
To avoid creating an extra table, it seems you could use a local test model without any trouble. `s.serialize([ScoreDecimal(score=decimal.Decimal(1.0))], cls=CustomJSONEncoder)`